src/mozillavpn.cpp

```diff
@@ -721,6 +721,12 @@ void MozillaVPN::errorHandle(ErrorHandler::ErrorType error) {
 
     case ErrorHandler::BackendServiceError:
       alert = BackendServiceErrorAlert;
+      if (m_userAuthenticated) {
+        deleteTasks();
+        // Deletes the controller-impl,Resets the status
+        // and spins up a new controller-impl
+        m_private->m_controller.initialize();
```

Comment:
OK, but what about all the other controller implementations?
We need to see if all of them work when calling initialize() again.